src/Command/RunCommand.php

```diff
@@ -28,6 +28,7 @@
 use Keboola\JobQueueInternalClient\Exception\StateTerminalException;
 use Keboola\JobQueueInternalClient\Exception\StateTransitionForbiddenException;
 use Keboola\JobQueueInternalClient\JobFactory;
+use Keboola\JobQueueInternalClient\JobFactory\Job;
```

Comment:
```suggestion
```

to je tady asi navic, mozna to vyuzujes v dlasich PRs

Reply:
Jo, to tam zůstalo z https://github.com/keboola/job-runner/pull/88, tam už to potřeba je. Tj. nechávám to tam.